website/docs/contributing/release.md

```diff
@@ -47,20 +47,26 @@ Take [Bump to version 0.36.0](https://github.com/apache/incubator-opendal/pull/2
 
 After bump version PR gets merged, we can create a GitHub release:
 
-- Create a tag at `main` branch on the `Bump Version` / `Patch up version` commit: `git tag -s "v0.36.0"`, please correctly check out the corresponding commit instead of directly tagging on the main branch.
+- Create a tag at `main` branch on the `Bump Version` / `Patch up version` commit: `git tag -s "v0.36.0-rc1"`, please correctly check out the corresponding commit instead of directly tagging on the main branch.
 - Push tags to GitHub: `git push --tags`.
```

Comment:
We should change our workflow to not push release for `rc` tags, except java.

Reply:
Yes. I'm not familiar with these part and I suggest @markt-asf we can approve the process and later update the workflow. Instead of blocking the grant of credential to build a solution without it XD